### DISCUSSION TITLE
Welcome to awesome_OpenSetRecognition_list Discussions!

### DISCUSSION THREAD

**Discussion by gary23ai** [iCGY96/awesome_OpenSetRecognition_list#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SikaStar** [comment]
Hi, I am Yongxing Dai from Peking University, Beijing, China. I am now focusing on Open Set Domain Adaptation.

**Comment by xyy19920105** [comment]
i'm a beignner of open set recognition. Could you tell me what is the differences between open set recognition and out of distribution detection?
